### PR TITLE
Fix example 1

### PR DESCRIPTION
**Pull request by Phlya**
Need to create a Tileset from the uuid, not just give the uuid to Track.

### CODE REVIEW

**Comment by flekschas** [comment]
Or use `tileset_uuid` instead of `tileset`

File: notebooks/Examples.ipynb [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "    #Track(track_type='top-axis', position='top'),\n",
     "    Track(track_type='heatmap', \n",
     "          position='center',\n",
-    "          tileset='CQMd6V_cRw6iCI_-Unl3PQ',\n",
+    "          tileset=higlass.tilesets.Tileset('CQMd6V_cRw6iCI_-Unl3PQ'),\n",
```
Comment (flekschas) [inline notebooks/Examples.ipynb]:
```suggestion
    "          tileset_uuid='CQMd6V_cRw6iCI_-Unl3PQ',\n",
```

**Comment by Phlya** [comment]
Thanks! Didn't know about `tileset_uuid` :)

**Comment by flekschas** [comment]
Thanks for finding the issue and the PR! :)

**Pull request merged**